### ISSUE TITLE
Loss is NaN

### ISSUE THREAD

**Issue by Mendel1** [kuangliu/pytorch-retinanet#9]
When I'm training the retinanet,cls_loss is  easy to become NaN.Does anyone know the reason?
```
loc_loss: 0.084 | cls_loss: nan | train_loss: nan | avg_loss: nan
```

**Comment by kuangliu** [comment]
See #6.
I'll update the loss soon.
